Description: Use decor plugin, decoration is gone.
 Compiz 0.9.x uses decor instead of decoration
Author: pi-rho <ubuntu@tyr.cx>
Bug-Ubuntu: https://launchpad.net/bugs/192389
Last-Updated: 2011-04-18

diff -Nur -x '*.orig' -x '*~' fusion-icon//FusionIcon/util.py fusion-icon.new//FusionIcon/util.py
--- fusion-icon//FusionIcon/util.py	2011-04-18 23:02:02.690623000 +0200
+++ fusion-icon.new//FusionIcon/util.py	2011-04-18 23:24:04.551805795 +0200
@@ -220,12 +220,12 @@
 
 		try:
 			context = compizconfig.Context( \
-				plugins=['decoration'], basic_metadata=True)
+				plugins=['decor'], basic_metadata=True)
 
 		except:
 			context = compizconfig.Context()
 
-		self.command = context.Plugins['decoration'].Display['command']
+		self.command = context.Plugins['decor'].Screen['command']
 
 		for decorator in installed.decorators:
 			self[decorator] = CompizDecorator(decorator, self, installed)
